worlds/alttp/__init__.py

```diff
@@ -136,6 +137,10 @@ class ALTTPWorld(World):
 
     create_items = generate_itempool
 
+    enemizer_path: str = Utils.get_options()["generator"]["enemizer_path"] \
+        if os.path.isabs(Utils.get_options()["generator"]["enemizer_path"]) \
+        else Utils.user_path(Utils.get_options()["generator"]["enemizer_path"])
```

Comment:
Two things:
1. (for me) the if is not required. os.join("something", "/an/abs/path") will return "/an/abs/path". I think i also tested it on windows some time ago, but you may want to double-check. I assume you copied that from "resolve_path", but at the time of writing that, i didn't know you could do what i described above.
2. it should be local_path, not user_path, if we assume it refers to the packaged enemizer. (This PR would currently break the AppImage.) We passed local_path to resolve_path for that reason.

Reply:
I copied that behaviour out of Generate, specifically so that no current assumptions should break...